torque-core/src/main/kotlin/com/workday/torque/TestChunkRetryer.kt

```diff
@@ -21,7 +23,11 @@ class TestChunkRetryer(private val adbDevice: AdbDevice,
                 var resultsResponse = testChunkRunner.run(args, testChunk)
                 while (resultsResponse.needToRetry(testChunk, args.retriesPerChunk)) {
                     adbDevice.log("Chunk has failed tests, retry count: ${testChunk.retryCount++}/${args.retriesPerChunk}, retrying...")
-                    resultsResponse = testChunkRunner.run(args, testChunk)
+                    resultsResponse = if (args.recordFailedTests && isLastRun(testChunk.retryCount, args.retriesPerChunk)) {
```

Comment:
What do you think about renaming to `shouldRecordFailedTests`?

Reply:
I've thought about it, but adopting this style will make a lot of boolean config options really long?